tests/integration/test_suite_hierarchy.py

```diff
@@ -0,0 +1,271 @@
+        {'name': child_suite_name, 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith(root_suite_name)},
+        {'name': test_name, 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith(child_suite_name)}
+    ]
+
+    verify_start_item_parameters(mock_client, expected_items)
+
+
+@mock.patch(REPORT_PORTAL_SERVICE)
+def test_simple_test_in_class_in_class(mock_client_init):
+    """Verify correct suite name and type for a test in a class in a class.
+
+    :param mock_client_init: Pytest fixture
+    """
+    root_suite_name = 'examples/hierarchy/test_in_class_in_class.py'
+    outer_child_suite_name = 'Tests'
+    inner_child_name = 'Test'
+    test_name = 'test_in_class_in_class'
+
+    mock_client = mock_client_init.return_value
+    mock_client.start_test_item.side_effect = item_id_gen
+
+    result = utils.run_pytest_tests(tests=[root_suite_name])
+    assert int(result) == 0, 'Exit code should be 0 (no errors)'
+
+    expected_items = [
+        {'name': root_suite_name, 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': outer_child_suite_name, 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith(root_suite_name)},
+        {'name': inner_child_name, 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith(outer_child_suite_name)},
+        {'name': test_name, 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith(inner_child_name)}
+    ]
+
+    verify_start_item_parameters(mock_client, expected_items)
+
+
+@mock.patch(REPORT_PORTAL_SERVICE)
+def test_simple_tests_in_different_inner_folders(mock_client_init):
+    """Verify correct suite hierarchy for two tests in different folders.
+
+    :param mock_client_init: Pytest fixture
+    """
+    tests_to_run = [
+        'examples/hierarchy/another_inner/test_another_inner_simple.py',
+        'examples/hierarchy/inner/test_inner_simple.py']
+    first_suite_name = 'examples/hierarchy/another_inner/' + \
+                       'test_another_inner_simple.py'
+    second_suite_name = 'examples/hierarchy/inner/test_inner_simple.py'
+    test_name = 'test_simple'
+
+    mock_client = mock_client_init.return_value
+    mock_client.start_test_item.side_effect = item_id_gen
+
+    result = utils.run_pytest_tests(tests=tests_to_run)
+    assert int(result) == 0, 'Exit code should be 0 (no errors)'
+
+    expected_items = [
+        {'name': first_suite_name, 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': test_name, 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith(first_suite_name)},
+        {'name': second_suite_name, 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': test_name, 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith(second_suite_name)}
+    ]
+
+    verify_start_item_parameters(mock_client, expected_items)
+
+
+TEST_VARIABLES = \
```

Comment:
Can you put these global variables below into a separate file? It's hard to concentrate on tests looking through the module. Thanks